pkg/networkservice/chains/client/client.go

```diff
@@ -51,22 +54,28 @@ func NewClient(ctx context.Context, clientOpts ...Option) networkservice.Network
 
 	return chain.NewNetworkServiceClient(
 		append(
-			[]networkservice.NetworkServiceClient{
-				adapters.NewServerToClient(setlogoption.NewServer(map[string]string{"name": opts.name})),
-				updatepath.NewClient(opts.name),
-				begin.NewClient(),
-				metadata.NewClient(),
-				opts.refreshClient,
-				clienturl.NewClient(opts.clientURL),
-				clientconn.NewClient(opts.cc),
-				dial.NewClient(ctx,
-					dial.WithDialOptions(opts.dialOptions...),
-					dial.WithDialTimeout(opts.dialTimeout),
-				),
-			},
+			append(
+				[]networkservice.NetworkServiceClient{
+					adapters.NewServerToClient(setlogoption.NewServer(map[string]string{"name": opts.name})),
+					updatepath.NewClient(opts.name),
```

Comment:
Why do we still have ServerToClient adapters here?  I would expect them to no longer be necessary.

Reply:
This commit is just workaround until `grpcfd` issue is done

Reply:
I don't understand how the ServerToClient works around the grpcfd issue... could you explain?

Reply:
I meant touched changes here.
As I can see, this PR doesn't add any new `ServerToClient`. This is what we are using in `main` branch https://github.com/networkservicemesh/sdk/blob/main/pkg/networkservice/chains/client/client.go#L55

Reply:
@Mixaster995 Why are we using the ServerToClient adapter in https://github.com/networkservicemesh/sdk/pull/1113#:~:text=https%3A//github.com/networkservicemesh/sdk/blob/main/pkg/networkservice/chains/client/client.go%23L55 ?

@glazychev-art We *really* should not be using ServerToClient outside of tests.  Why is it being used as part oc healing here?

Reply:
It is not part of healing. We just touched these lines (spaces, tabulation...) so we see changes here.
Ok, got it. Probably we need to create `setlogoption.NewClient` instead of `adapters.NewServerToClient`

Reply:
@edwarnicke it is not required to use adapters and those completely unrelated to heal. It was part of PR for improving logs and a temporary solution for this chain element and i will remove adapters as soon as i can.

Reply:
@Mixaster995 Many thanks :)